models/_sector/dex/trades/ethereum/platforms/solidly_v3_ethereum_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'solidly_v3_ethereum',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v3_trades(
+        blockchain = 'ethereum',
+        project = 'solidly',
+        version = '3',
+        Pair_evt_Swap = source('solidly_ethereum', 'SolidlyV3Pool_evt_Swap'),
```

Comment:
Ideally to be consistent I would like to use solidly_v3_ethereum.SolidlyV3Pool_evt_Swap here,
But contract is not accepted as solidly_ethereum contract already exists.